libs/traits/src/fees.rs

```diff
@@ -0,0 +1,90 @@
+use frame_support::{dispatch::DispatchResult, traits::tokens::Balance};
+use parity_scale_codec::FullCodec;
+use scale_info::TypeInfo;
+use sp_runtime::traits::MaybeSerializeDeserialize;
+
+/// Type used for identity the key used to retrieve the fees.
+pub trait FeeKey:
+	FullCodec + TypeInfo + MaybeSerializeDeserialize + sp_std::fmt::Debug + Clone + PartialEq
+{
+}
+
+impl<
+		T: FullCodec + TypeInfo + MaybeSerializeDeserialize + sp_std::fmt::Debug + Clone + PartialEq,
+	> FeeKey for T
+{
+}
```

Comment:
Formatting seems off

Reply:
The line is not small enough to not fit in the same line as `<` but not big enough to be split in several lines. 😆